pkg/concurrency/timeout.go

```diff
@@ -29,10 +29,13 @@ func Timeout(duration time.Duration) WaitableChan {
 //
 // select {
 // case <-sig.Done():
-//   ... happy path ...
+//
+//	... happy path ...
+//
 // case <-TimeoutOr(timeoutDuration, &sig):
-//   ... oh no, a timeout ...
-// }
+//
+//	  ... oh no, a timeout ...
+//	}
```

Comment:
Should the closing bracket be aligned with `select` and `case` ?
Should there be an empty comment line above ?